tests/integration/conftest.py

```diff
@@ -95,7 +96,10 @@ def token(pytestconfig: pytest.Config) -> str:
     """Configured token setting."""
     token = pytestconfig.getoption("--token")
     assert token, "Please specify the --token command line option"
-    return token
+    secondary_tokens: str = pytestconfig.getoption("--secondary-tokens", default="")
+    tokens = secondary_tokens.split(",").append(token)
```

Comment:
AFAIK `append` doesn't return a list. Consider 
```
tokens = secondary_tokens.split(",")
tokens.append(token)
```
instead

Reply:
You're very right, thanks for catching that, i've added additional parsing logic!